staging/src/k8s.io/component-base/feature/feature_gate.go

```diff
@@ -25,9 +25,13 @@ var (
 	// Only top-level commands/options setup and the k8s.io/component-base/featuregate/testing package should make use of this.
 	// Tests that need to modify feature gates for the duration of their test should use:
 	//   defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.<FeatureName>, <value>)()
-	DefaultMutableFeatureGate featuregate.MutableFeatureGate = featuregate.NewFeatureGate()
+	DefaultMutableFeatureGate featuregate.MutableFeatureGate = featuregate.NewFeatureGateWithInitializer(initializeFeaturesFromEnvironmentVariables)
```

Comment:
component-base should not hold an authoritative global of featuregates.  Those are are logically owned by the components that import component-base.